### PR TITLE
NoteFrequency: Update to full template

### PR DESCRIPTION
**Pull request by lamanh**
Updates NoteFrequency Re: #2767

Maintainer: @sublinear
https://duck.co/ia/view/note_frequency

### CODE REVIEW

**Comment by daxtheduck** [comment]
@lamanh Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/NoteFrequency.pm [hunk 1]
```diff
@@ -64,9 +64,13 @@ handle remainder => sub {
     # result
     return $frequency,
         structured_answer => {
-            input => [html_enc($letter.$accidental.$octave." in A".$tuning." tuning")],
-            operation => "Note Frequency",
-            result => html_enc($frequency." Hz"),
+            data => {
+                title    => html_enc($frequency." Hz"),
```
Comment (moollaza) [inline lib/DDG/Goodie/NoteFrequency.pm]: we don't need to `html_enc` $frequency -- it's just a number we're creating so there's no worry of unescaped HTML content being there

**Comment by edgesince84** [comment]
@lamanh,  thanks again for fixing the Note Frequency IA!  You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

File: lib/DDG/Goodie/NoteFrequency.pm (hunk 1 above)
Comment (lamanh) [inline lib/DDG/Goodie/NoteFrequency.pm]: @moollaza, updated.

**Comment by GuiltyDolphin** [comment]
@lamanh Thanks! :dolphin:

**Pull request merged**